src/update_hexes.py

```diff
@@ -384,10 +390,14 @@ def one_function_to_rule_them_all(common_info: CommonInfo, specific_info: Specif
     '''
 
     print('Getting AGOL references...')
-    gis = arcgis.gis.GIS(common_info.portal, common_info.username)
+    password = getpass('Enter Password: ')
```

Comment:
**question**: this is ok because you watch this task run right?

Reply:
correct- it's a manual process for now. Someday it may be automated, in which case I'll do proper password management.